certbot/cli.py

```diff
@@ -591,6 +592,11 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):
              "certificates. Updates to the Subscriber Agreement will still "
              "affect you, and will be effective 14 days after posting an "
              "update to the web site.")
+    helpful.add(
+        None, "--update-registration", action="store_true",
```

Comment:
You should make a "register" help topic, and use that for this argument, rather than `None`.  Search for "paths" or "automation" to see how to do that.